### ISSUE TITLE
Support project-based config dir

### ISSUE THREAD

**Issue by moltar** [apifytech/apify-cli#115]
Use case: I am juggling multiple client accounts and I need to be able to push to different accounts.

Currently only one account is supported via `GLOBAL_CONFIGS_FOLDER`.

https://github.com/apifytech/apify-cli/blob/cfbd6566f7c41eb5ce0b6e3e74aa1e96970b1d13/src/lib/consts.js#L100

Ideally it should allow for project-based config dir (e.g. `.../project/.apify/*`), and/or allow having multiple accounts under the homedir (e.g. `$HOME/.apify/acc1`).

Inspiration can be taken from `aws-cli` or `kubectl`.

**Comment by uiii** [comment]
any progress on this?

**Comment by mnmkng** [comment]
No, sorry. But it's not completely dead, we're just completely reworking the way this and many other things will be handled. For reference see [the Actor specs](https://github.com/apify/actor-specs)

**Comment by patrikbraborec** [comment]
Closing — consolidated into #445, the single tracker for multi-account/profile support (named profiles + `--profile` flag, with the auth file redesign from #567 as prerequisite).
